packages/lib/src/components/Popconfirm/index.tsx

```diff
@@ -2,8 +2,8 @@ import { useRef, useState } from "react";
 import styled from "@emotion/styled";
 import Button from "@leafygreen-ui/button";
 import Tooltip, { TooltipProps } from "@leafygreen-ui/tooltip";
-import { size, zIndex } from "../../constants/tokens";
-import { wordBreakCss } from "../styles";
+import { size, zIndex } from "constants/tokens";
+import { wordBreakCss } from "components/styles";
```

Comment:
Components in the lib directory that are imported by spruce or parsley will still need relative imports to work. Only tests and stories that aren't imported elsewhere will work with absolute imports